src/expr/array_store_all.cpp

```diff
@@ -96,4 +104,4 @@ size_t ArrayStoreAllHashFunction::operator()(const ArrayStoreAll& asa) const {
   return TypeHashFunction()(asa.getType()) * ExprHashFunction()(asa.getExpr());
 }
 
-} /* CVC4 namespace */
+}  // namespace CVC4
```

Comment:
We should be consistent here in terms of comment style, use /* */. We should change that in the clang format file.

Reply:
SG. I'm happy to use that as long as we can automate it.